packages/color-wheel/src/color-wheel.css

```
@@ -19,3 +19,62 @@ governing permissions and limitations under the License.
 :host(:focus) {
     outline: none;
 }
+
+.wheel {
+    width: 100%;
+    height: 100%;
+    background: conic-gradient(
+        from 90deg,
+        rgb(255, 0, 0),
+        rgb(255, 128, 0),
+        rgb(255, 255, 0),
+        rgb(128, 255, 0),
+        rgb(0, 255, 0),
+        rgb(0, 255, 128),
+        rgb(0, 255, 255),
+        rgb(0, 128, 255),
+        rgb(0, 0, 255),
+        rgb(128, 0, 255),
+        rgb(255, 0, 255),
+        rgb(255, 0, 128),
+        rgb(255, 0, 0)
+    );
+}
+
+.wheel:before,
+.wheel:after {
+    content: '';
+    position: absolute;
+    border-color: var(--spectrum-colorwheel-border-color);
+    border-width: var(
+        --spectrum-colorwheel-border-size,
+        var(--spectrum-alias-border-size-thin)
+    );
+    border-radius: 50%;
+    border-style: solid;
+}
+
+.wheel:after {
+    inset: 0;
+}
+
+.wheel:before {
+    inset: 24px;
+}
+
+:host([disabled]) .wheel:before,
+:host([disabled]) .wheel:after {
+    /* .spectrum-ColorWheel.is-disabled .spectrum-ColorWheel-innerCircle,
+   * .spectrum-ColorWheel.is-disabled .spectrum-ColorWheel-outerCircle */
+    border-color: var(
+        --spectrum-colorwheel-border-color-disabled,
+        var(--spectrum-global-color-gray-300)
+    );
+}
+:host([disabled]) .wheel {
+    /* .spectrum-ColorWheel.is-disabled .spectrum-ColorWheel-outerCircle */
+    background: var(
```

```suggestion
    background-color: var(
```

Would it make sense to include more specificity here in the property name to prevent shorthand input?  Or if someone were to add an image to `--spectrum-colorwheel-fill-color-disabled` an okay use of the hook?

---

I can't put a `conic-gradient` into `background-color`, can I? This value needs to override the `conic-gradient` that applied the hue visuals by default.

Also, great question about `--spectrum-colorwheel-fill-color-disabled` being given an image. If it was given an image, regardless of its use in `background` or `background-color`, the results would be somewhat unexpected. Not sure there's safety there outside of a future where we have typed custom properties.

---

Ah good point, that function is creating an image. `background-image` then maybe? https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/conic-gradient()#gradient_at_40-degrees

Though I see a lot of the examples just use the shorthand so I'm sure it's not a problem.

---

Cool. I'll leave as `background` for now but these are good lines of inquiry to make time for with each change we make to the library!